src/main/java/org/folio/helper/PurchaseOrderHelper.java

```diff
@@ -512,6 +509,65 @@ private CompletableFuture<Void> setPoNumberIfMissing(CompositePurchaseOrder comp
     return completedFuture(null);
   }
 
+  private CompletionStage<Void> setPoNumberPrefix(CompositePurchaseOrder compPO) {
```

Comment:
I feel `setPoNumberPrefix` and `setPoNumberSuffix` contain logic that is not substantial enough validate generating a future. These methods could live on the model. It wouldn't stick out if these methods returned a simple `void`.

Reply:
i agree, fixed